CDH-Studio/UpSkill
# removed sidebar, made temp cards look better

##What does this PR do?
improves the styling of result cards

## relevant jiras
jira.ised-isde.canada.ca/browse/STUDIOUP-6
jira.ised-isde.canada.ca/browse/STUDIOUP-37

##steps to test
make first name last name search. Make sure cards fill up rows before columns , profile picture has been added and skill tags now are contained in labels.

##note
Result cards are still very WIP because we are just showing some data from GEDS right now. This is more for planning/proof of concept right now
